packages/iris-grid/src/IrisGridRenderer.ts

```diff
@@ -151,25 +158,93 @@ class IrisGridRenderer extends GridRenderer {
     column: VisibleIndex,
     row: VisibleIndex
   ): void {
-    const { metrics, model } = state;
-    const { modelColumns, modelRows } = metrics;
-    const modelRow = getOrThrow(modelRows, row);
+    const { metrics } = state;
+    const { modelColumns } = metrics;
     const modelColumn = modelColumns.get(column);
     if (modelColumn === undefined) {
       return;
     }
-    const value = model.valueForCell(modelColumn, modelRow);
-    if (TableUtils.isTextType(model.columns[modelColumn]?.type)) {
-      if (value === null || value === '') {
-        const originalFont = context.font;
-        context.font = `italic ${originalFont}`;
-        const displayValue = value === null ? 'null' : 'empty';
-        super.drawCellContent(context, state, column, row, displayValue);
-        context.font = originalFont;
-        return;
-      }
+
+    const cellRenderer = this.getCellRenderer('text');
+    cellRenderer.drawCellContent(context, state, column, row);
```

Comment:
Should be getting the cell renderer type from the model, then getting the renderer and using that